packages/reports/src/Containers/open-positions.tsx

```diff
@@ -490,7 +488,7 @@ const OpenPositions = ({
     const [accumulator_rate, setAccumulatorRate] = React.useState(accumulator_rates[0]);
     const is_accumulator_selected = contract_type_value === contract_types[2].text;
     const is_multiplier_selected = contract_type_value === contract_types[1].text;
-    const show_accu_in_dropdown = !is_eu && is_virtual;
+    const show_accu_in_dropdown = !is_eu;
```

Comment:
@henry-deriv nit-pick :) we also can remove this line, and simply rename it when we destructure props:
L466: `is_eu: show_accu_in_dropdown,`